Issue 99: Informative messaging for priors

athowes opened the pull request:
<!-- Thanks for opening this pull request! Below we have provided a suggested template for PRs to this repository and a checklist to complete before opening a PR -->
 
## Description

This PR closes #99.

## Checklist

- [ ] My PR is based on a package issue and I have explicitly linked it.
- [ ] I have included the target issue or issues in the PR title in the for Issue(s) *issue-numbers*: PR title
- [ ] I have read the [contribution guidelines](https://github.com/epinowcast/.github/blob/main/CONTRIBUTING.md).
- [ ] I have tested my changes locally.
- [ ] I have added or updated unit tests where necessary.
- [ ] I have updated the documentation if required.
- [ ] My code follows the established coding standards.
- [ ] I have added a news item linked to this PR.
- [ ] I have reviewed CI checks for this PR and addressed them as far as I am able.

<!-- Thanks again for this PR - @epinowcast dev team -->

seabbs commented on R/latent_individual.R:
```diff
@@ -205,12 +205,21 @@ epidist_family.epidist_latent_individual <- function(data, family = "lognormal",
   prior2 <- brms::prior("normal(0, 0.5)", class = "Intercept", dpar = "sigma")
+
+  cli::cli_inform(c(
+    "i" = "The following priors have been set:",
```
I think you need to be what the link function is for each or this is confusing.

athowes replied:
I have written something corresponding more directly to what the `brms` prior is. I think that we still have some confusion about the priors and families which we will need to handle (e.g. #131, #171). So I'd be in favour of this PR just saying what there is at the moment, and it's something we can come back to once we sort out how we are handling families.

seabbs reviewed (commented):
I'm not sure I would want this level of verbosity but I am happy to go with it if you think its needed. One small issue with needing to define what the link is in order for people to understand what they are looking at.

athowes commented:
Yes I also think it's a bit verbose. What about an option `verbose = FALSE` and have it only show if set to `TRUE`?

seabbs commented:
> Yes I also think it's a bit verbose

We should be able to use`cli` so it only shows once per session or once every 8 hours etc. Perhaps that is the way to go?

athowes commented:
Give message about prior on the transformed more natural scale.

athowes commented:
I've added an issue about the additional complexities here: https://github.com/epinowcast/epidist/issues/172

I've added the "display once every 8 hours". It's also possible to do "display on new session" instead if people have a preference.
